cypress/e2e/verifyLogin.cy.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types= "cypress" />
+
+describe('Validate', function(){
+    
+    before(()=> {
+        cy.login(Cypress.env())
```

Comment:
Fix the Login command, remove the parameters in the function, and update the existing tests to not include any parameter.
Like:
```
cy.login();
```